howto/stream/debug.md

```diff
@@ -0,0 +1,36 @@
+Debugging issues that occur when streaming your application can be tricky. The following instructions give you some pointers on how to start tracking down streaming issues.
+
+First of all, check the error message that occurs. Unfortunately, it is often hard to determine why a connection fails, which is why the error message does not always give a clear indication of the source of the error. Typical errors are covered though, so in these cases, the error message should give you an idea on where to look.
```

Comment:
I think we can do a lot better in the SDK to generate more valuable error messages. Going to plan work for this.

Reply:
From the developer's perspective, they can use `experimental.debug=false` option which would yield out debug information and help to diagnose session info as well E.g. sdp info. 
(I'm not quite sure if it's the right place to put it here.)

Reply:
@adglkh Where would you set this option?

Reply:
With the following example.
https://github.com/anbox-cloud/discourse-docs/blob/2764c5620f5350052db99b968f0a784a7aa660be/howto/stream/web-client.md
People can enable the debug logging through

```
    let stream = new AnboxStream({
        connector: connector,
        targetElement: "anbox-stream",
        controls: {
            keyboard: true
        },
+       experimental: {
+           debug: true,
+       },
        callbacks: {
            error: error => {
                console.log("AnboxStream failed: ", error);
            }
        }
    });
    stream.connect();
    </script>
```

Reply:
As of now, it's impossible. We had a quick discussion about this just a little moment ago, we're going to support this in the dashboard, this task will be prioritized by @edlerd and @morphis . As an outcome, people can enable it in the stream view.

Reply:
Thanks, I've added the information about experimental.debug to the web-client page.